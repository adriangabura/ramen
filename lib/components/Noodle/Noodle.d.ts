/// <reference types="react" />
declare function Noodle(): JSX.Element;
export default Noodle;
