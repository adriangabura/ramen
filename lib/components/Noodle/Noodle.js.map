{"version":3,"file":"Noodle.js","sourceRoot":"","sources":["../../../src/components/Noodle/Noodle.tsx"],"names":[],"mappings":";;;;;;;;;AAAA,2CAA+B;AAE/B,SAAS,MAAM;IACb,OAAO,CAAC,0CAAiB,CAAC,CAAC;AAC7B,CAAC;AAED,kBAAe,MAAM,CAAC"}