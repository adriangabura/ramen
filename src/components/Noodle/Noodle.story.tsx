import { storiesOf } from "@storybook/react";
import * as React from "react";

import Noodle from "./Noodle";

storiesOf("Noodle", module)
  .add("simple noodle", () => <Noodle />);
