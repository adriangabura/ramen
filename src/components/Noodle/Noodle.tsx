import * as React from "react";

function Noodle() {
  return (<div>noodle</div>);
}

export default Noodle;
