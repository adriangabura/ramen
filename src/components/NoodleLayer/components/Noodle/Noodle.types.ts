import { IGraphConnection } from "../../../../redux/connections/connections.types";
import { IPosition } from "../../../../types";

export interface INoodleProps {
  id: number;
  start: IPosition;
  end: IPosition;
  onPointerDown?: () => void;
}
