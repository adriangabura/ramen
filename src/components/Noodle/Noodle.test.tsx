import React from "react";
import renderer from "react-test-renderer";

import Noodle from "./Noodle";

describe("Noodle", () => {

  it("renders without crashing", () => {
    const tree = renderer.create(<Noodle />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
