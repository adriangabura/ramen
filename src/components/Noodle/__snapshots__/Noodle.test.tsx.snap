// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Noodle renders without crashing 1`] = `
<div>
  noodle
</div>
`;
